src/c++/perf_analyzer/data_loader.cc

```diff
@@ -324,6 +324,7 @@ DataLoader::GetInputData(
     const uint8_t** data_ptr, size_t* batch1_size)
 {
   bool data_found = false;
+  *data_ptr = nullptr;
```

Comment:
added for consistency. GetOutputData already does this. The caller shouldn't need to set it to nullptr before calling this.